test/templates/identifier/component.njk

```diff
@@ -1,11 +0,0 @@
-<!--
```

Comment:
shall we remove the whole folder ?

Reply:
you said in another comment to bring back the identifier tests?